app/api/test_attempts_api.rb

```diff
@@ -0,0 +1,201 @@
+require 'grape'
+
+class TestAttemptsApi < Grape::API
+  format :json
+
+  helpers AuthenticationHelpers
+  helpers AuthorisationHelpers
+
+  before do
+    authenticated?
+  end
+
+  # Handle common exceptions
+  rescue_from :all do |e|
```

Comment:
These are handled by the API at the moment... so we should be able to remove these